docs/validator-tickets-calculation.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Validator Tickets Calculation
+slug: /protocol/validator-tickets-calculation
+---
+
+:::info 
+This page is a guide to help you understand how Validator Tickets (VTs) are calculated.
+:::
+
+### VT Consumption Calculation Summary
+
+The VT (Validator Tickets) consumption calculation is a critical process for managing node operators within the Puffer Protocol. This process ensures that node operators maintain sufficient VT balances to support their active and exiting validators. Here's a breakdown of how VT consumption is calculated:
+
+1. **Node Operator Address Validation**: The process begins by validating the node operator's address to ensure it is in the correct format.
+
+2. **Fetching VT Balance**: The current VT balance of the node operator is retrieved from the Puffer Protocol. This balance represents the total VTs available for consumption by the node operator's validators.
+
+3. **Epoch and Validator Data Retrieval**: The latest epoch is fetched from the beacon chain, along with the list of active validators and those that have exited but not yet finalized. Active validators consume VTs daily, while exited validators may still consume VTs until they are fully finalized.
```

Comment:
maybe explain `finalized` somehow, somewhere. 
Those are the validators that exited beacon chain, but the process has not been fully completed and the user did not receive the bond yet, something like that but more simple maybe